src/ApiNavigationList.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+	<NcAppNavigationList :aria-labelledby="arialabel">
+		<NcAppNavigationItem v-for="app in apps"
+			:key="app.id"
+			allow-collapse
+			:name="app.name"
+			:open="isSelectedChild($route.params.appid, app)"
+			:to="{path: `/view/${app.id}`}">
+			<template #icon>
+				<AppIcon v-if="app.icon_url"
+					:href="app.icon_url" />
+				<ApiIcon v-else />
+			</template>
+			<NcAppNavigationItem v-for="child in childApis(app)"
+				:key="child.id"
+				:name="child.name"
```

Comment:
I wonder if we should strip the app id prefix so that it's just `full` and not `provisioning_api-full`. I think this would look a bit better.